### PR TITLE
Obsolete this repository

### PR DESCRIPTION
**Pull request by natemcmaster**
Follow up to #50.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+Platform Abstractions (obsolete)
+================================
 
-Travis: [![Build Status](https://travis-ci.org/aspnet/PlatformAbstractions.svg?branch=dev)](https://travis-ci.org/aspnet/PlatformAbstractions)
+This component only shipped in ASP.NET Core 1 and has been removed from ASP.NET Core 2 and up.
 
-Platform abstractions for applications.
+### Replacing API usage
 
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
```
Comment (Eilon) [inline README.md]: Please keep this line.

**Review by Eilon — APPROVED** [review]
Approved w/ one change.

File: README.md (hunk 1 above)
Comment (natemcmaster) [reply README.md]: Ok. Re-added.

**Comment by natemcmaster** [comment]
Btw, @Eilon you can disable AppVeyor for this repo.

**Comment by Eilon** [comment]
> Btw, @Eilon you can disable AppVeyor for this repo.

@natemcmaster I disabled Travis and AppVeyor. Thanks!

**Pull request merged**